inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -661,7 +667,8 @@ public void onRecommendationAcceptEvent(AjaxRecommendationAcceptedEvent aEvent)
         Optional<AnnotationObject> oRecommendation = model.getPredictionByVID(document, vid);
 
         if (!oRecommendation.isPresent()) {
-            LOG.error("Could not find annotation in [{}] with id [{}]", document, vid);
+            LOG.error("Could not find prediction in [{}] with id [{}]", document, vid);
+            error("Could not find prediction");
```

Comment:
When logging an error to the UI within an AJAX request, it must be ensured that any FeedbackPanel present on the page is also re-rendered. Applies to other `error("...")` calls as well.

Reply:
How would I achieve that?

Reply:
```
aEvent.getTarget().addChildren(getPage(), IFeedback.class);
```